opentelemetry-tracing/opentelemetry-java-tracing/okhttp-and-jedis/src/main/java/com/splunk/tracing/otel/examples/javaagent/App.java

```diff
@@ -75,16 +77,13 @@ private void doFetchAndSet(String url) {
                 System.out.println("Error: " + e);
                 return;
             }
-
             executor.submit(() -> redisClient.set(url, respBody));
         } catch (Throwable e) {
             span.recordException(e);
-            System.out.println(e);
-        } finally {
-            span.end();
+            span.setStatus(StatusCode.ERROR, e.getMessage());
```

Comment:
I kinda expected that `recordException` already does that?

Reply:
I also wanted/expected that, but lost that fight in the specs. Instead, we just record exception-related attributes and an event. I guess the idea is that exceptions might be recorded without it being an error in the actual span.